src/builder/index.ts

```diff
@@ -32,6 +33,11 @@ export async function buildSW(
     );
   }
   code = `importScripts('${importFrom}')\n`;
-  code += `AMP_SW.init(${serializeObject(config || {})})`;
+  if (eject) {
+    code += 'AMP_SW.installNoOpServiceWorker()';
+  } else {
+    code += `AMP_SW.init(${serializeObject(config || {})})`;
+  }
+
   return code;
 }
```

Comment:
Let's move this all to testing directories, since it's purely for test cases.

Reply:
Will do this in a separate PR, issue opened #30